src/wp-includes/block-editor.php

```diff
@@ -409,19 +409,20 @@ function wp_get_first_block( $blocks, $block_name ) {
  * Retrieves Post Content block attributes from the current post template.
  *
  * @since 6.3.0
+ * @since 6.4.0 return null by default or an empty array when the Post Content block exists but has no attributes.
```

Comment:
I think we can simplify this to only explain the `null` change. The other part ("empty array when the Post Content block exists but has no attributes") was already like that before I'd argue.
```suggestion
 * @since 6.4.0 Return null if there is no post content block.
```